src/server/scripts/Kalimdor/zone_azuremyst_isle.cpp

```diff
@@ -51,6 +51,7 @@ enum draeneiSurvivor
     SAY_THANK_FOR_HEAL     = 0,
     SAY_ASK_FOR_HELP       = 1,
     SPELL_IRRIDATION       = 35046,
+    SPELL_GIFT_OF_THE_NAARU = 59544,
```

Comment:
each class has its own "Gift of the Naaru" spell

Reply:
Well, i dont know why but it work with every class.